test/components/generators/test_hf_utils.py

```diff
@@ -42,3 +42,23 @@ def test_additional_accepted_params_unknown_parameter():
     # Although strange_param is not generation param the check_generation_params
     # does not raise exception because strange_param is passed as additional_accepted_params
     check_generation_params(kwargs, additional_accepted_params)
+
+
+@pytest.mark.integration
```

Comment:
@silvanocerza any idea why this marker is being ignored? Tests fails on CI because they think the test is unit test

Reply:
It's not this one that is failing, it's `test_generate_text_response_with_valid_prompt_and_generation_parameters` in the `TestHuggingFaceTGIGenerator` class.

https://github.com/deepset-ai/haystack/actions/runs/7785504824/job/21228519135#step:5:1164

Reply:
Actually all tests in `TestHuggingFaceTGIGenerator` I think. 🤔

Reply:
O wow, let me look into this